app/src/laos/java/utils/PhoneMask.java

```diff
@@ -0,0 +1,52 @@
+package utils;
+
+/**
+ * Created by ina on 02/08/2016.
+ */
+public class PhoneMask {
+
+    /**
+     * Formatted telephone mask: (xxx) xxx-xxx = 9 (020) xxxx-xxxx = 11  (030) xxx-xxxx = 10
+     */
+    public static final String FORMATTED_PHONENUMBER_MASK = "(\\(020\\) \\d{4}-\\d{3})|(\\(030\\) \\d{3}-\\d{4})|(\\(\\d{3}\\) \\d{3}-\\d{3})";
+
+    /**
+     * Formatted telephone mask: xxxxxxxxx = 9 020xxxxxxxx = 11  030xxxxxxx = 10
+     */
+    public static final String PLAIN_PHONENUMBER_MASK = "(((020)\\d{8})|((030)\\d{7})|\\d{9})";
+
+    /**
+     * Formats number according to mask (xxx) xxx-xxx = 9 (020) xxxx-xxxx = 11  (030) xxx-xxxx = 10
+     * @param phoneValue
+     * @return
+     */
+    public static String formatPhoneNumber(String phoneValue) {
+        //Empty -> nothing to format
+        if (phoneValue == null || "".equals(phoneValue)) {
```

Comment:
@idelcano Do we need this `"".equals(phoneValue)`? Probably yes but I do not understand why...